plugins/experimental/txn_box/plugin/src/ip_space.cc

```diff
@@ -907,10 +907,10 @@ Mod_ip_space::operator()(Context &ctx, feature_type_for<IP_ADDR> addr)
   }
   Feature value{FeatureView::Literal("")};
   if (active._space) {
-    auto &&[range, payload] = *active._space->space.find(addr);
-    active._row             = range.empty() ? nullptr : &payload;
-    active._addr            = addr;
-    active._drtv            = drtv;
+    auto [range, payload] = *active._space->space.find(addr);
```

Comment:
This will call the copy constructor for both the range and payload. If `&&` doesn't work, does `&` work? That will avoid a copy.

Reply:
let me try more variations on this.

Reply:
the documentation of swoc/IPRange.h const_iterator with the auto && [r, p] seems to indicate that the compiler should take care of the danging references.  Apparently that's not always true.